docs/source/posts/2022/2022-08-17-week-09-shivam.rst

```diff
@@ -0,0 +1,31 @@
+Week 9 - First working skeletal animation prototype
+===================================================
+.. post:: August 16 2022
```

Comment:
The date mentioned in the filename and this line are different.
Maybe this won't affect it much but still, I think it would be better to keep the same.